Commands.md

```diff
@@ -408,6 +409,54 @@ build
 *.log
 ```
 
+## git ignore-io
+
+Generate sample gitignore file from [gitignore.io](https://www.gitignore.io)  
+
+Without option, `git ignore` show the sample gitignore on screen.  
```

Comment:
`git ignore` shows global gitignore if option not given.
`git ignore-io` prints usage if option not given.